src/utils/forms-loader.test.ts

```diff
@@ -246,6 +255,15 @@ describe('Forms loader - getForm', () => {
                   ],
                   id: 'dateTestPerformed',
                 },
+                {
+                  id: 'sampleQuestion',
+                  label: 'Question title',
+                  questionOptions: {
+                    concept: '193e8a03-1581-46e9-b0a4-17c55fa2649f',
+                    rendering: 'text',
+                  },
+                  type: 'obs',
+                },
```

Comment:
Ditto

Reply:
<img width="898" alt="Screenshot 2024-10-18 at 17 38 57" src="https://github.com/user-attachments/assets/0dd62b3c-8b8f-43c1-b47f-99b89f95ebb3">


Removing this block brings this error